__Main__.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>Please never make another one of these plots lol</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-Farhad1353/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>OK</p>